### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by JusticeRage**
This pull request includes the changes suggested in #24's review.

### CODE REVIEW

**Comment by JusticeRage** [comment]
Thanks! I'm all in favor of a proofreading.

**Comment by eschaefer** [comment]
@hugogiraudel would you be up for a quick French language proofread? 😺

**Comment by HugoGiraudel** [comment]
Sure.

File: locales/fr.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+fr:
+  site_name: "Sécurité sans Frontières"
+  follow_us: "Suivez-nous sur Twitter : %{twitter}"
+  request_assistance: "Demander de l'aide"
+  intro_paragraphs: "Depuis l'avènement de l'ère digitale, ceux qui dénoncent
+    la corruption, luttent contre le racisme et défendent la justice et la
+    démocratie sur Internet sont plus que jamais menacés.<br /><b>Sécurité sans
```
Comment (HugoGiraudel) [inline locales/fr.yml]: Not sure the markup makes sense here.

File: locales/fr.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  activist_headline: "Vous êtes un activiste, un journaliste ou un défenseur des
+    droits de l'homme ?"
+  activist_p1: "Sécurité sans Frontières est à votre service. Nous pouvons vous
+    assister en conduisant un audit de sécurité de votre site internet, en menant
+    des investigations en cas d'attaque, et de manière générale en vous 
+    conseillant sur les problématiques liées à la cybersécurité. Les prestations de 
+    sécurisation étant le plus souvent coûteuses, tout individu ou organisation qui
+    lutte contre les violations des droits de l'homme, le racisme et autres 
+    injustices peut bénéficier du soutien de SSF à titre gracieux."
+  activist_p2: "Contactez nouz en remplissant le formulaire accessible via le
```
Comment (HugoGiraudel) [inline locales/fr.yml]: “Contactez-nous”

File: locales/fr.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+fr:
+  site_name: "Sécurité sans Frontières"
+  follow_us: "Suivez-nous sur Twitter : %{twitter}"
+  request_assistance: "Demander de l'aide"
```
Comment (HugoGiraudel) [inline locales/fr.yml]: It should be a curly quote. Same for all the quotes in this document.

Comment (JusticeRage) [inline locales/fr.yml]: But... There are no quotes in the whole document, except the YAML ones!

File: locales/fr.yml (hunk 1 above)
Comment (JusticeRage) [inline locales/fr.yml]: Not sure either, but it's consistent with the original text. If it's changed here, it should be changed everywhere.

File: locales/fr.yml (hunk 2 above)
Comment (JusticeRage) [inline locales/fr.yml]: Agreed.

File: locales/fr.yml (hunk 3 above)
Comment (JusticeRage) [inline locales/fr.yml]: Ack.

File: locales/fr.yml (hunk 1 above)
Comment (eschaefer) [inline locales/fr.yml]: I will change this after the PR, since it needs to be changes in all translations

**Pull request closed without merging**